exercises/isbn-verifier/example.go

```diff
@@ -0,0 +1,59 @@
+package isbn
+
+import (
+	"math"
+	"strconv"
+)
+
+func IsVaildISBN(isbn string) bool {
+
+	// step 1
+	isbn = dropHyphen(isbn)
+
+	// step 2
+	ary := strToAry(isbn)
+
+	if len(ary) != 10 {
+		return false
+	}
+
+	// step 3
+	return calcCheckDigit(ary)
+
+}
+
+func dropHyphen(isbn string) string {
+	var result string
+	for _, char := range isbn {
+		if char == '-' {
+			continue
+		}
+		result += string(char)
+	}
+	return result
+}
+
+func strToAry(isbn string) []int {
+	var result []int
+	for _, char := range isbn {
+		if char == 'X' {
+			result = append(result, 10)
+		} else {
+			s := string(char)
+			i, _ := strconv.Atoi(s)
```

Comment:
There is an assumption here, it returns default (0) on error. 

IMHO, we should handle it to return early or ensure value is zero.